### DISCUSSION TITLE
Progress bar to track transcription

### DISCUSSION THREAD

**Discussion by mriamnobody** [openai/whisper#1665] [Ideas]
Dear Devs,

I am deeply fond of the Whisper model; it has been incredibly helpful in addressing almost all of my needs with ease. I'm a big fan of it. However, one aspect of the model does give me a moment of concern: the lack of a progress bar or status indicator during the transcription process. This becomes particularly noticeable when transcribing larger files, those boasting around 50-60 hours of runtime. The integration of a progress or status bar could greatly alleviate this issue. Is it possible to incorporate this feature? If guidance is available, I am willing to take on the task myself. Although I'm a newcomer with limited knowledge of machine learning or programming, I am eager to give it my best shot.  Thank you.

Edit: I'm using the model locally on my machine.

**Comment by glangford** [comment]
Have you tried transcribing with `--verbose False` ? There is a progress bar displayed if that option is selected.

https://github.com/openai/whisper/blob/0a60fcaa9b86748389a656aa013c416030287d47/whisper/transcribe.py#L221
